packages/core/src/node/utils/normalizePath.ts

```diff
@@ -41,9 +41,9 @@ function absolutePathToRoutePrefix(
     ),
   );
   const routeService = RouteService.getInstance();
-  const [basePrefix, versionPrefix, langPrefix] =
+  const [versionPrefix, langPrefix, _] =
```

Comment:
```suggestion
  const [versionPrefix, langPrefix] =
```

I don't think `_` is needed here?